cli/src/cli_util.rs

```diff
@@ -613,9 +611,7 @@ impl WorkspaceCommandHelper {
             base: workspace.workspace_root().clone(),
         };
         let helper = Self {
-            string_args: command.string_args.clone(),
-            global_args: command.global_args.clone(),
-            settings,
```

Comment:
nit: `settings` no longer has to be cloned above

Reply:
Obsolete

Reply:
This is still valid.
```diff
@@ -614,7 +614,7 @@
         repo: Arc<ReadonlyRepo>,
         loaded_at_head: bool,
     ) -> Result<Self, CommandError> {
-        let settings = command.data.settings.clone();
+        let settings = &command.data.settings;
         let commit_summary_template_text =
             settings.config().get_string("templates.commit_summary")?;
         let revset_aliases_map =
```
(or inline it)

Reply:
Sorry, fixed now. I also went through and replaced a bunch of `&command.data.settings` by `command.settings()` and similar.